pkg/localmetrics/localmetrics.go

```diff
@@ -113,7 +114,7 @@ func UpdateAWSMetrics(kubeClient kubeclientpkg.Client, hour int) {
 
 	d := time.Duration(hour) * time.Hour
 	for range time.Tick(d) {
-		accountTotal, err := account.TotalAwsAccounts(awsClient)
+		accountTotal := totalawsaccountwatcher.TotalAccountWatcher.awsLimit
```

Comment:
I think at this point, since we already have this functionality in the totalaccountwatcher, this function can be removed, and we can put this line:
`MetricTotalAWSAccounts.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(accountTotal))`

on line 96 of the `pkg/totalawsaccountwatcher/totalawsaccountwatcher.go` file

Reply:
Will do